src/reservedKeywords.ts

```diff
@@ -0,0 +1,35 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+
+type Data = {
+    keyword: string;
+    required: boolean;
+};
```

Comment:
```suggestion
type Keyword = {
    <value|name>: string;
    required: boolean;
};
```
I suggest this small change for the Keyword type, what do you think?